client.go

```diff
@@ -98,6 +98,12 @@ func NewClient(ctx context.Context, options ...func(Party) error) (Client, error
 			}
 		}
 	}
+	// Wrap logging with timestamps
```

Comment:
could we please remove these again? They produce duplicate timestamps if using any other logging lib. Adding the timestamp should imho be task of the user-chosen logging lib, not of the log-generating code.

Reply:
IMHO the timestamp should be taken when the event occurs. This is vital if you have a distributed system.
go-kit/log is a structured logger, so it should be simple to strip the ts key/value pair if you like to add a timestamp yourself.